addons/viewport/src/preview/index.js

```diff
@@ -0,0 +1,20 @@
+import addons from '@storybook/addons';
+import { ADD_VIEWPORTS_EVENT_ID, SET_VIEWPORTS_EVENT_ID } from '../shared';
+
+export function init() {}
```

Comment:
What is this for?

Reply:
I couldn't find a clear API documentation describing the preview API's, so I decided to follow the same approach as manager

Reply:
Got it, but this method is a bit misleading, so let's remove it.

Reply:
I removed it